### DISCUSSION TITLE
Newb - can't get this working :(

### DISCUSSION THREAD

**Discussion by BuffaloBrennan** [prasathmani/tinyfilemanager#1113] [General]
All I get when running the tinyfilemanager.php is a blank screen :(  I'm obviously doing something wrong, but I haven't a clue.

I changed the $auth_users  but still nothing.

Thank you in advance for any help.

**Comment by prasathmani** [comment]
can you check in php error log.

**Comment by BuffaloBrennan** [comment]
I'm using php 8.1.24 on Windows IIS.  The Error log is completely empty and the reponse from the page is blank (well 2 blank lines)

**Reply by prasathmani** [reply]
https://learn.microsoft.com/en-us/iis/application-frameworks/install-and-configure-php-applications-on-iis/using-fastcgi-to-host-php-applications-on-iis

https://www.experts-exchange.com/questions/26881055/PHP-IIS-Blank-page.html

**Reply by BuffaloBrennan** [reply]
Thanks for the response.  However, I have FastCGI already install and working.  I've written a very large H&S database system using PHP, Bootstrap, jquery, jquery-ui, etc.   I'm investigating changing our current file manager & photo viewer for a better one.

If I put in echos and html near the top of tinyfilemanager.php that comes though fine .

**Reply by prasathmani** [reply]
@BuffaloBrennan , are you embedding the TFM into other PHP file ?

https://github.com/prasathmani/tinyfilemanager/wiki/Embedding

**Reply by BuffaloBrennan** [reply]
I "Viewed the Source" for the online demo page and used the Embedding:
```
<?php
define('FM_EMBED', true);
define('FM_SELF_URL', $_SERVER['PHP_SELF']);
require '/php/tinyfilemanager.php';
?>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1, shrink-to-fit=no"
    />
    <meta
      name="description"
      content="Web based File Manager in PHP, Manage your files efficiently and easily with Tiny File Manager"
    />
    <meta name="author" content="CCP Programmers" />
    <meta
      name="keywords"
      content="tinyfilemanager, php filemanager, filemanager, ftp, file-explore, file-manager, upload-files"
    />
    <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
    <title>Tiny File Manager | H3K | Demo</title>
  </head>
  <body>
    <iframe
      src="http://tinyfilemanager.alwaysdata.net/index.php?p="
      style="
        position: fixed;
        top: 0px;
        left: 0px;
        bottom: 0px;
        right: 0px;
        width: 100%;
        height: 100%;
        border: none;
        margin: 0;
        padding: 0;
        overflow: hidden;
        z-index: 999999;
      "
    >
      Your browser doesn't support iframes
    </iframe>
  </body>
</html>
```
